pyproject.toml

```diff
@@ -1,10 +1,7 @@
-[build-system]
-requires = ["flit"]
-build-backend = "flit.buildapi"
-
-[tool.flit.metadata]
-module = "orchestrator"
-dist-name = "orchestrator-core"
+[project]
+name = "orchestrator-core"
+version = "0.3.5"
+description = "Open source orchestration softwre for NREN's"
```

Comment:
Typo: "Opensource orchestration software"

Reply:
Ah yes, good spot. I corrected "software" but when I google "opensource" then I see it's mostly written with a space in between. So I left the space there for now.